tests/components/august/test_lock.py

```diff
@@ -80,6 +80,5 @@
     await _create_august_with_devices(hass, lock_details)
 
     lock_brokenid_name = hass.states.get("lock.brokenid_name")
-    # Once we have bridge_is_online support in py-august
-    # this can change to STATE_UNKNOWN
-    assert lock_brokenid_name.state == STATE_UNAVAILABLE
+
+    assert lock_brokenid_name.state == STATE_UNKNOWN
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5072309)